pkgs/top-level/all-packages.nix

```diff
@@ -3022,7 +3022,7 @@ with pkgs;
   ### APPLICATIONS/TERMINAL-EMULATORS
 
   alacritty = callPackage ../applications/terminal-emulators/alacritty {
-    inherit (darwin.apple_sdk.frameworks) AppKit CoreGraphics CoreServices CoreText Foundation OpenGL;
+    inherit (darwin.apple_sdk_11_0.frameworks) AppKit CoreGraphics CoreServices CoreText Foundation OpenGL;
```

Comment:
I’m surprised this works (because usually it results in mixing headers between SDK versions), but it builds for me.

If I want to use `overrideSDK`, I have to override `rustPlatform`, which is considerably uglier.

Reply:
I would say let's leave it at that for now and reconsider if we see problems appearing. Thank you for the comment!